frontend/src/modules/S3_Datasets/views/DatasetCreateForm.js

```diff
@@ -494,45 +502,103 @@ const DatasetCreateForm = (props) => {
                       <Card>
                         <CardHeader title="Governance" />
                         <CardContent>
-                          <TextField
-                            fullWidth
-                            error={Boolean(
-                              touched.SamlGroupName && errors.SamlGroupName
+                          <Autocomplete
+                            id="SamlAdminGroupName"
+                            disablePortal
+                            options={groupOptions.map((option) => option)}
+                            onChange={(event, value) => {
+                              if (value && value.value) {
+                                setFieldValue(
+                                  'SamlAdminGroupName',
+                                  value.value
+                                );
+                              } else {
+                                setFieldValue('SamlAdminGroupName', '');
+                              }
+                            }}
+                            inputValue={values.SamlAdminGroupName}
+                            renderInput={(params) => (
+                              <Box>
+                                {groupOptions.length > 0 ? (
+                                  <TextField
+                                    {...params}
+                                    fullWidth
+                                    error={Boolean(
+                                      touched.SamlAdminGroupName &&
+                                        errors.SamlAdminGroupName
+                                    )}
+                                    helperText={
+                                      touched.SamlAdminGroupName &&
+                                      errors.SamlAdminGroupName
+                                    }
+                                    label="Team"
+                                    variant="outlined"
```

Comment:
nit: for here and all other related TextField componenets changed in commit a094fbca8d643cf63ea1d18020e8630bbf07de15 - removing the `onChange={handleChange}` option now no longer allows typing in the textfield 

Would add back unless that option was causing some issue with updating groupOptions and such

Reply:
Reverted back, great catch